app/front/scripts/services/downloader.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
this is service for downloading files or making API requests. It supports caching too. 
And if we want to change `fetch` to something other, we can do it only in one place.